Cartfile.resolved

```diff
@@ -0,0 +1,3 @@
+binary "https://assets.braintreegateway.com/mobile/ios/carthage-frameworks/cardinal-mobile/CardinalMobile.json" "2.2.5-2-xcframework-only-update"
+binary "https://assets.braintreegateway.com/mobile/ios/carthage-frameworks/pp-risk-magnes/PPRiskMagnes.json" "5.4.0"
+binary "https://github.com/paypal/paypalcheckout-ios/raw/main/Carthage/PayPalCheckout.json" "0.94.0"
```

Comment:
It looks like on master we don't have a cartfile resolved, which makes sense, but wondering if there is a specific reason we need it now or if it can be removed.

Reply:
Currently, in the CI step where we run `carthage build --no-skip-current`, we were not fetching the remote dependencies, and would fail that step. This did not impact Magnes / Cardinal, since those binaries were already stored on our repo in the `Frameworks` directory. Checking in the `Cartfile.resolved` allows that build step to fetch the remote dependencies. 

Carthage also recommends you check that file in - https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#cartfileresolved